lang/hr/index.md

```diff
@@ -0,0 +1,262 @@
+1. MANJU verziju kada dodate unatrag-kompatibilne funkcionalnosti i
+1. ZAKRPA verziju kada napravite unatrag-kompatibilne ispravke bugova.
+
+Dodatne oznake za pred-izdanja i metapodatke builda mogu se koristiti kao proširenje na VEĆA.MANJA.ZAKRPA format.
+
+Uvod
+------------
+
+U svijetu upravljanja softverom postoji strašno mjesto koje zovemo
+"pakao ovisnosti." Što više vaš sustav raste i što više paketa
+integrirate u vaš softver, vjerojatnije je da ćete se, jednoga dana,
+naći u toj jami očaja.
+
+U sustavima s više ovisnosti, izdavanje novih verzija paketa, može se vrlo brzo
+pretvoriti u noćnu moru. Ako su specifikacije ovisnosti previše stroge, postoji
+opasnost zaključavanja verzije (nemogućnosti nadogradnje paketa bez potrebe
+izdavanja nove verzije svakog ovisnog paketa). Ako su pak ovisnosti specificirane
+previše slobodno, sigurno ćete osjetiti posljedice verzijskog promiskuiteta
+(pretpostavke kompatibilnosti s više budućih verzija nego što je razumno).
+Pakao ovisnosti je slučaj gdje zaključavanje verzije i/ili verzijski promiskuitet
+sprječavaju lagano i sigurno napredovanje projekta.
+
+Kao rješenje tog problema predlažem jednostavan set pravila i zahtjeva
+koji će nalagati kako se dodjeljuju i povećavaju oznake verzija.
+Ova se pravila temelje, no nisu nužno i ograničena, na već postojećoj
+i raširenoj uobičajenoj praksi u projektima otvorenog i zatvorenog koda.
+Kako bi ovaj sustav funkcionirao, potrebno je prvo objaviti javni API. Možemo
+to primijeniti u dokumentaciji ili u samom kodu. U svakom slučaju, važno je da
+API bude jasan i precizan. Jednom kad identificiramo javni API, komuniciramo
+promjene u njemu kroz specificirana povećanja u oznaci/broju verzije. Uzmimo
+format verzije X.Y.Z (MANJA.VEĆA.ZAKRPA). Ispravci bugova koji ne utječu na API
+povećavaju oznaku zakrpe, unatrag-kompatibilni dodaci/promjene u APIju
+povećavaju manju verziju, a unatrag inkompatibilne promjene u APIju povećavaju
+veću verziju.
+
+Ovaj sam sustav nazvao "Semantičko verzioniranje". Unutar ove sheme, oznake
+verzije i način na koji se mijenjaju prenose određene informacije o kodu koji
+se nalazi ispod i što se mijenjalo od jedne do druge verzije.
+
+
+Specifikacija semantičkog verzioniranja (SemVer)
+------------------------------------------
+
+Ključne riječi "MORA", "NE SMIJE", "POTREBNO", "BITI ĆE", "NEĆE BITI", "TREBALO BI",
+"NE BI TREBALO", "PREPORUČENO", "MOŽE", i "OPCIONALNO" ("MUST", "MUST NOT",
+"REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY",
+"OPTIONAL") u ovom dokumentu treba interpretirati kao što je opisano u
+[RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+1. Softver koji koristi Semantičko verzioniranje MORA objaviti javni API. API
+može biti deklariran u samom kodu ili postojati samo unutar dokumentacije.
+U svakom slučaju, mora biti precizan i jasan.
+
+1. Normalna oznaka verzije MORA biti u formatu X.Y.Z gdje su X, Y i Z prirodni
```

Comment:
Natural ("prirodni") numbers are positive integers, i.e. they don't include 0. This is not the same as non-negative integers which include 0. The correct translation would be something like "ne-negativni cijeli brojevi".

Reply:
this was a big mistake on my part. will change